src/NBitcoin/IBitcoinSerializable.cs

```diff
@@ -72,14 +73,16 @@ public static void FromBytes(this IBitcoinSerializable serializable, byte[] byte
             serializable.ReadWrite(bms);
         }
 
-        public static T Clone<T>(this T serializable, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION, Network network = null) where T : IBitcoinSerializable, new()
+        public static T Clone<T>(this T serializable, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION, Network network = null) where T : IBitcoinSerializable
```

Comment:
I suggest creating a new method `FactoryClone<T>()` without a `new()`
And throw if `instance == null`